### ISSUE TITLE
java.lang.StringIndexOutOfBoundsException exception when executing exportAirr

### ISSUE THREAD

**Issue by vincentwalter** [milaboratory/mixcr#1388]
## Expected Result

I want to export sequences to AIRR format. This works flawlessly for 57/58 of my samples.

## Actual Result

With one sample, an error is reproducibly thrown and the resulting tsv is truncated.

## Exact MiXCR commands 

`mixcr exportAirr -f sample.clns sample.tsv`

## MiXCR report files 
```Exporting to AIRR format: 0%
Exporting to AIRR format: 59.8%  ETA: 00:00:00
Please copy the following information along with the stacktrace:
   Version: 4.5.0; built=Fri Sep 22 12:39:05 UTC 2023; rev=cdb24b4fb7; lib=repseqio.v3.0.1
        OS: Linux
      Java: 17.0.8.1
  Cmd args: exportAirr -f sample.clns sample.tsv
picocli.CommandLine$ExecutionException: Error while running command exportAirr java.lang.StringIndexOutOfBoundsException: begin -1, end 1, length 17
        at com.milaboratory.mixcr.cli.Main.registerExceptionHandlers$lambda-12(SourceFile:340)
        at picocli.CommandLine.execute(CommandLine.java:2088)
        at com.milaboratory.mixcr.cli.Main.main(SourceFile:98)
Caused by: java.lang.StringIndexOutOfBoundsException: begin -1, end 1, length 17
        at java.base/java.lang.String.checkBoundsBeginEnd(String.java:4606)
        at java.base/java.lang.String.substring(String.java:2709)
        at com.milaboratory.o.ke$c.a(SourceFile:2189)
        at com.milaboratory.mixcr.cli.CommandExportAirr.run1(SourceFile:319)
        at com.milaboratory.mixcr.cli.MiXCRCommandWithOutputs.run0(SourceFile:69)
        at com.milaboratory.mixcr.cli.MiXCRCommand.run(SourceFile:37)
        at picocli.CommandLine.executeUserObject(CommandLine.java:1939)
        at picocli.CommandLine.access$1300(CommandLine.java:145)
        at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2358)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2352)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:2314)
        at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2179)
        at picocli.CommandLine$RunLast.execute(CommandLine.java:2316)
        at com.milaboratory.mixcr.cli.Main.registerLogger$lambda-26(SourceFile:447)
        at picocli.CommandLine.execute(CommandLine.java:2078)
        ... 1 more```

**Comment by mizraelson** [comment]
Hi, will it be possible to share the `.clns` file for this sample?

**Comment by vincentwalter** [comment]
Sure. Can I send it by email?

**Comment by mizraelson** [comment]
Of course, or Google Drive if its large. 
support@milaboratories.com

Sincerely,
Mark

**Comment by mizraelson** [comment]
Fixed in #1406
